src/main/java/mort/task/Task.java

```diff
@@ -0,0 +1,98 @@
+package mort.task;
+
+/**
+ * Class that represents a task.
+ */
+public class Task {
```

Comment:
Should this perhaps be an abstract class, since we might not be able to create a Task without knowing what type it is?